DependencyInjection/HttplugExtension.php

```diff
@@ -152,6 +156,9 @@ private function configurePluginByName($name, Definition $definition, array $con
                 $definition->replaceArgument(0, new Reference($config['journal']));
                 break;
             case 'logger':
+                if (class_exists('Http\Client\Common\Plugin\LoggerPlugin')) {
```

Comment:
What about using `LoggerPlugin::class` and add a proper `use` statement (same with the other places)?